packages/cfpb-forms/src/organisms/Multiselect.js

```diff
@@ -416,6 +417,20 @@ function Multiselect( element ) { // eslint-disable-line max-statements
       }
     } );
 
+    _headerDom.addEventListener( 'mouseup', function( event ) {
+      const target = event.target;
+
+      /* Check if we're over the down-arrow on the right side of the input.
+         Also check if the fieldset is open.
+         35 = width of the arrow on the right of the search input.
+         140 = the max-height value set in multiselect.less for the fieldset.
+      */
+      if ( event.layerX > target.offsetWidth - 35 &&
+           _fieldsetDom.offsetHeight === 140 ) {
```

Comment:
Is this the best way we have to check that the fieldset is open?

Reply:
Yeah I know it's hackery :( I think we need to do some re-writing of the drop-down. It opens when the search input gets focus and closes when it's blurred. Focus happens on mousedown, so it gets the attributes of being open when it is clicked, not when the drop-down has actually completed opening. Getting the height was the only way I found to reliably find that it was actually open. We should re-write this to dispatch events for the life cycle of the drop-down expand/collapse and then listen to the `expandEnd` event (or actually probably the `transitionEnd` event) and add attributes at that point, which this if statement would then query. As noted in this PR, the collapse method is what dispatches (erroneously I think) the `expandEnd` event.

Reply:
Makes sense. This is fine for now, but could you capture everything you've said here in a new issue? (Or do we already have one open?)

Reply:
https://github.com/cfpb/design-system/issues/1182